lerobot/common/robots/robot.py

```diff
@@ -23,7 +23,9 @@ def __init__(self, config: RobotConfig):
         self.robot_type = self.name
         self.id = config.id
         self.calibration_dir = (
-            config.calibration_dir if config.calibration_dir else HF_LEROBOT_CALIBRATION / ROBOTS / self.name
+            config.calibration_dir
+            if config.calibration_dir
+            else Path(HF_LEROBOT_CALIBRATION / ROBOTS / self.name)
```

Comment:
This can stay as it was since `HF_LEROBOT_CALIBRATION` is also already a `Path`
```suggestion
            config.calibration_dir if config.calibration_dir else HF_LEROBOT_CALIBRATION / ROBOTS / self.name
```